Packs/Base/ReleaseNotes/1_13_11.md

```diff
@@ -0,0 +1,3 @@
+#### Scripts
+##### CommonServerPython
+- Fixed an issue where a proxy configuration without the 'http://' prefix failed some integrations.
```

Comment:
```suggestion
- Fixed an issue where a proxy configuration without the 'http://' prefix would cause some integrations to fail.
```

Reply:
@yuvalbenshalom @amshamah419  I would go with a different approach regarding the release notes:

```suggestion
- Added support to auto append `http://` protocol to proxy urls configured without a protocol. This resolves issues seen where recent versions of urllib3 failed parsing proxy urls.
```